app/controllers/transcriptions_controller.rb

```diff
@@ -17,15 +17,16 @@ def show
     @transcription = Transcription.find(params[:id])
     authorize @transcription
 
+    if TranscriptionPolicy.new(current_user, @transcription).has_update_rights? &&
+       !@transcription.locked?
+      @transcription.lock!(current_user)
```

Comment:
The formatting looks kind of ugly here, any suggests on how to format better?

Reply:
how's about

models/transcription.rb:
```
def unlocked?
  !locked?
end
```

controllers/transcriptions_controller.rb#show:
```
if policy.has_update_rights? && @transcription.unlocked?
  @transcription.lock!(current_user)
end

private

def policy
  @policy ||= TranscriptionPolicy.new(current_user, @transcription)
end
```

Reply:
niiice. that'd clean it up.